apps/src/code-studio/components/progress/StageLock.jsx

```diff
@@ -10,18 +10,23 @@ import { openLockDialog, closeLockDialog, lockStage } from '../../stageLockRedux
 import { stageShape } from './types';
 
 const styles = {
+  lockSettings: {
+    ...progressStyles.whiteButton,
+    maxWidth: 240,
+    overflow: 'hidden'
+  },
   lockSettingsText: {
     marginLeft: 10
   },
   warning: {
-    position: 'relative',
-    top: 2
+    marginBottom: 5,
```

Comment:
to prove I actually read all of this, my only nit is that there are there are a couple of extra commas in the styles objects